CODE_OF_CONDUCT.rst

```diff
@@ -4,80 +4,128 @@ Contributor Covenant Code of Conduct
-separately.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+graphql-compiler-maintainer@kensho.com.
+All complaints will be reviewed and investigated promptly and fairly.
+
+All community leaders are obligated to respect the privacy and security of the
+reporter of any incident.
+
+Enforcement Guidelines
+----------------------
+
+Community leaders will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct:
+
+1. Correction
+^^^^^^^^^^^^^
+
+**Community Impact**: Use of inappropriate language or other behavior deemed
+unprofessional or unwelcome in the community.
+
+**Consequence**: A private, written warning from community leaders, providing
+clarity around the nature of the violation and an explanation of why the
+behavior was inappropriate. A public apology may be requested.
+
+2. Warning
+^^^^^^^^^^
+
+**Community Impact**: A violation through a single incident or series
+of actions.
+
+**Consequence**: A warning with consequences for continued behavior. No
+interaction with the people involved, including unsolicited interaction with
+those enforcing the Code of Conduct, for a specified period of time. This
+includes avoiding interactions in community spaces as well as external channels
+like social media. Violating these terms may lead to a temporary or
+permanent ban.
+
+3. Temporary Ban
+^^^^^^^^^^^^^^^^
+
+**Community Impact**: A serious violation of community standards, including
+sustained inappropriate behavior.
+
+**Consequence**: A temporary ban from any sort of interaction or public
+communication with the community for a specified period of time. No public or
+private interaction with the people involved, including unsolicited interaction
+with those enforcing the Code of Conduct, is allowed during this period.
+Violating these terms may lead to a permanent ban.
+
+4. Permanent Ban
+^^^^^^^^^^^^^^^^
+
+**Community Impact**: Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior,  harassment of an
+individual, or aggression toward or disparagement of classes of individuals.
 
-Project maintainers who do not follow or enforce the Code of Conduct in
-good faith may face temporary or permanent repercussions as determined
-by other members of the project's leadership.
+**Consequence**: A permanent ban from any sort of public interaction within
+the community.
 
 Attribution
 -----------
 
 This Code of Conduct is adapted from the `Contributor
-Covenant <http://contributor-covenant.org>`__, version 1.4, available at
-`http://contributor-covenant.org/version/1/4 <http://contributor-covenant.org/version/1/4/>`__
+Covenant <http://contributor-covenant.org>`__, version 2.0, available at
+`http://contributor-covenant.org/version/2/0 <http://contributor-covenant.org/version/2/0/>`__
```

Comment:
A small nice-to-have: the Contributor Covenant page is serving HSTS headers with `max-age=31536000` so I feel it's a pretty safe bet that we can make the URL be HTTPS by default.
```suggestion
`https://contributor-covenant.org/version/2/0 <https://contributor-covenant.org/version/2/0/>`__
```

Reply:
Good catch, thanks!